src/components/ResultsTable.tsx

```diff
@@ -1,45 +1,121 @@
+                            const contactData = result.owner_details.find((owner) => owner.Contact_Type === 'Director')
+                            return (
+                                <tr key={result.id}>
+                                    <td>{index + 1}</td>
+                                    <td>{propertyAddress}</td>
+                                    <td>{ownerData?.Name || ''}</td>
+                                    <td>{contactData?.Name || 'Contact Is Not Found'}</td>
+                                </tr>
+                            )
+                        })}
+                    </tbody>
+                </table>
+            </div>
+                )
+            }
+            {props.filterInUse === 'SalesEvidenceFilter' &&
+             (
+                <div style={{ padding: '20px' }}>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Address</th>
+                            <th>Land Area</th>
+                            <th>Build Area</th>
+                            <th>Date Sold</th>
+                            <th>Sale Price</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {props.results.map((result) => {
+                            let propertyAddress = result.Deal_Name
+                            const landArea = result.Land_Area_sqm
+                            const buildArea = result.Build_Area_sqm
+                            const dateSold = result.Sale_Date
+                            const salePrice = result.Sale_Price
+                            return (
+                                <tr key={result.id}>
+                                    <td>{propertyAddress}</td>
+                                    <td>{landArea}</td>
+                                    <td>{buildArea}</td>
+                                    <td>{dateSold}</td>
+                                    <td>{salePrice}</td>
+                                </tr>
+                            )
+                        })}
+                    </tbody>
+                </table>
+            </div>
+                )
+            }
+            {props.filterInUse === 'LeaseFilter' &&
+             (
+                <div style={{ padding: '20px' }}>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Address</th>
+                            <th>Land Area</th>
+                            <th>Build Area</th>
+                            <th>Date Sold</th>
+                            <th>Sale Price</th>
+                        </tr>
+                    </thead>
+                        <tbody>
+                        {props.results.map((result) => {
+                            let propertyAddress = result.Property.find((address) => address.name)
```

Comment:
These don't need to be variablised. The `SalesEvidenceFilter` ones don't need to be either. But they'll be ignored when I rebased my branch with master.